src/main/resources/db/migration/V6__INTRODUCE_UUID_DATA_TYPE.sql

```diff
@@ -0,0 +1,21 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
Still need this?

Reply:
Unfortunately, yes I think so. Online resources were a bit unclear about this and I'm still not sure if it's not needed when one uses the Postgres DATA TYPE UUID or just for creation of UUIDs. More on this below...